packages/web-components/fast-btr/src/fixtures/attribute/attribute.spec.ts

```diff
@@ -0,0 +1,26 @@
+import { expect, test } from "@playwright/test";
+
+test.describe("f-template", async () => {
+    test("create a non-binding attribute", async ({ page }) => {
+        await page.goto("/attribute");
+
+        const customElementInput = await (await page.locator("test-element")).locator("input");
```

Comment:
locators themselves don't have to be awaited:

```suggestion
        const customElement = page.locator("test-element");
        const customElementInput = customElement.locator("input");
```